### DISCUSSION TITLE
how to read effective date to a airflow dag from run time config in airflow

### DISCUSSION THREAD

**Discussion by shwar123** [apache/airflow#29310] [Q&A]
Hi Team,

Can anyone help on "how to read effective date to a airflow dag from a run time config in airflow"

Regards,
Eshwar

**Comment by potiuk** [comment]
I think you have to be more specific than that. The way how to convert the schedule parameter depends on which timetable you euse: https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/timetable.html#timetables - the documentation is there and date depends on the schedule chosen. Read those and related docs. 

This is just a generic response you can have, to generic question you have. But if you have a specific question after reading the docs - maybe you can ask them and get a bit more specific answer.

**Reply by shwar123** [reply]
Thank You  Potiuk!, let mw check the docs
